src/ol/source/wmtssource.js

```diff
@@ -481,3 +533,125 @@ ol.source.WMTS.optionsFromCapabilities = function(wmtsCap, config) {
   /* jshint +W069 */
 
 };
+
+
+/**
+ * Return the GetFeatureInfo URL for the passed coordinate, resolution, and
+ * projection. Return `undefined` if the GetFeatureInfo URL cannot be
+ * constructed.
+ * @param {ol.Coordinate} coordinate Coordinate.
+ * @param {number} resolution Resolution.
+ * @param {ol.proj.Projection} projection Projection.
+ * @param {!Object=} opt_params Optional GetFeatureInfo params.
```

Comment:
The params object. It's passed in by the user right? So we need to have an externs for this as well. Using object['property'] notation is only a workaround IMHO.